doc/classes/Image.xml

```diff
@@ -376,12 +376,12 @@
 			<argument index="2" name="color" type="Color">
 			</argument>
 			<description>
-				Sets the [Color] of the pixel at [code](x, y)[/code] if the image is unlocked. Example:
+				Sets the [Color] of the pixel at [code](x, y)[/code] if the image is locked. Example:
 				[codeblock]
 				var img = Image.new()
-				img.lock()
-				img.set_pixel(x, y, color) # Does not have an effect
 				img.unlock()
```

Comment:
I think unlock should be after lock, or it might be confusing to the readers IMO. You usually want to lock, do something, then finally unlock.